src/build/buildApp.js

```diff
@@ -110,19 +109,14 @@ function maybeCopyScripts(srcs, dest) {
   });
 }
 
-function normalizeAppName(appName, url) {
-  // use a simple 3 byte random string to prevent collision
-  const hash = crypto.createHash('md5');
-  hash.update(url);
-  const postFixHash = hash.digest('hex').substring(0, 6);
-  const normalized = _.kebabCase(appName.toLowerCase());
-  return `${normalized}-nativefier-${postFixHash}`;
+function normalizeAppName(appName) {
+  return _.kebabCase(`${appName.toLowerCase()}`);
```

Comment:
You can remove the backticks and ${} here.

Reply:
@mark-carlson you marked the PR as approved. Did you want this addressed before it's merged? @0xnu if you have a minute, might as well do this now in case @mark-carlson wants that for the PR.

Reply:
@voltechs, a very minor point.  It's good to go as long as the tests pass  :) . If this get's merged, we can close the one I opened.